x-pack/plugins/observability/public/components/app/section/apm/index.tsx

```diff
@@ -0,0 +1,125 @@
+import { onBrushEnd } from '../helper';
+import { StyledStat } from '../../styled_stat';
+
+interface Props {
+  startTime?: string;
+  endTime?: string;
+  bucketSize?: string;
+}
+
+function formatTransactionValue(value?: number) {
+  return numeral(value).format('0.00a');
+}
+
+export const APMSection = ({ startTime, endTime, bucketSize }: Props) => {
+  const theme = useContext(ThemeContext);
+  const history = useHistory();
+
+  const { data, status } = useFetcher(() => {
+    if (startTime && endTime && bucketSize) {
+      return getDataHandler('apm')?.fetchData({ startTime, endTime, bucketSize });
+    }
+  }, [startTime, endTime, bucketSize]);
+
+  const { title = 'APM', appLink, stats, series } = data || {};
+
+  const min = moment.utc(startTime).valueOf();
+  const max = moment.utc(endTime).valueOf();
+
+  const formatter = niceTimeFormatter([min, max]);
+
+  const isLoading = status === FETCH_STATUS.LOADING;
+
+  const transactionsColor = series?.transactions.color || theme.euiColorVis1;
+
+  return (
+    <SectionContainer
+      minHeight={296}
+      title={title || 'APM'}
+      subtitle={i18n.translate('xpack.observability.overview.chart.apm.subtitle', {
+        defaultMessage: 'Summary',
+      })}
+      appLink={appLink}
+      hasError={status === FETCH_STATUS.FAILURE}
+    >
+      <EuiFlexGroup>
+        <EuiFlexItem grow={false}>
+          <EuiStat
+            title={numeral(stats?.services.value).format('0a')}
+            description={stats?.services.label || ''}
+            titleSize="s"
+            isLoading={isLoading}
+          />
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <StyledStat
+            title={formatTransactionValue(stats?.transactions.value)}
+            description={stats?.transactions.label || ''}
+            titleSize="s"
+            isLoading={isLoading}
+            color={transactionsColor}
+          />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+      <ChartContainer height={177} isLoading={isLoading}>
+        <Chart size={{ height: 177 }}>
+          <Settings
+            onBrushEnd={({ x }) => onBrushEnd({ x, history })}
+            theme={theme.darkMode ? DARK_THEME : LIGHT_THEME}
+            showLegend={false}
+            xDomain={{ min, max }}
+          />
+          {series?.transactions.coordinates && (
+            <>
+              <BarSeries
+                id="transactions"
+                name="Transactions"
+                data={series?.transactions.coordinates}
+                xScaleType={ScaleType.Time}
+                yScaleType={ScaleType.Linear}
+                xAccessor={'x'}
+                yAccessors={['y']}
+                color={transactionsColor}
+              />
+              <Axis
+                id="y-axis"
+                position={Position.Left}
+                showGridLines
+                tickFormat={(value) => `${numeral(value).format('0.00a')} tpm`}
```

Comment:
```suggestion
                tickFormat={(value) => `${formatTransactionValue(value)} tpm`}
```

Reply:
With the above change this would be:

```suggestion
                tickFormat={(value) => `${formatTransactionsPerMinute(value)} tpm`}
```